support/ci/deploy_mac_launcher.sh

```diff
@@ -26,7 +26,8 @@ else
 fi
 
 # first update the copy of the habitat code stored on the mac server to the latest
-ssh ${ssh_args} ${mac_builder} << EOF
+# shellcheck disable=2087
```

Comment:
```
In ./support/ci/deploy_mac_launcher.sh line 29:
ssh "${ssh_args}" ${mac_builder} << EOF
                                    ^-- SC2087: Quote 'EOF' to make here document expansions happen on the server side rather than on the client.
```
I assume this is correct and we want `$TRAVIS_BUILD_NUMBER` expanded client-side, but `$HOME` expanded server side.